src/routes/search/components/SearchModal.js

```diff
@@ -88,7 +88,7 @@ class SearchModal extends React.Component<PropsType, StateType> {
     return <ScrollView contentContainerStyle={{ flexGrow: 1 }}>
       <SpaceBetween>
         <View><CategoryList categories={filteredCategories} query={query} onItemPress={this.onItemPress}
-                            theme={theme} language={language} /></View>
+                            theme={theme} language={language} thumbnail={''} /></View>
```

Comment:
```suggestion
                            theme={theme} language={language} /></View>
```
Would be cooler, if we could just omit `thumbnail` if it's not necessary. (Can be done using @klinzo's suggestions)